src/webportal/src/app/job/job-view/fabric/task-attempt.jsx

```diff
@@ -0,0 +1,143 @@
+import StatusBadge from '../../../components/status-badge';
+import { getDurationString } from '../../../components/util/job';
+
+const params = new URLSearchParams(window.location.search);
+const userName = params.get('username');
+const jobName = params.get('jobName');
+const jobAttemptIndex = params.get('jobAttemptIndex');
+const taskRoleName = params.get('taskRoleName');
+const taskIndex = params.get('taskIndex');
+
+const TaskAttemptPage = () => {
+  const [loading, setLoading] = useState(true);
+  const [taskStatus, setTaskStatus] = useState(null);
+
+  const getTimeDuration = (startMs, endMs) => {
+    const start = startMs && DateTime.fromMillis(startMs);
+    const end = endMs && DateTime.fromMillis(endMs);
+    if (start) {
+      return Interval.fromDateTimes(start, end || DateTime.utc()).toDuration([
+        'days',
+        'hours',
+        'minutes',
+        'seconds',
+      ]);
+    } else {
+      return null;
+    }
+  };
+
+  useEffect(() => {
+    fetchTaskStatus(
+      userName,
+      jobName,
+      jobAttemptIndex,
+      taskRoleName,
+      taskIndex,
+    ).then(data => {
+      setTaskStatus(data);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <div>
+      {loading && <SpinnerLoading />}
+      {!loading && (
+        <Stack styles={{ root: { margin: '30px' } }} gap='l1'>
+          <div>
+            <ActionButton
+              iconProps={{ iconName: 'revToggleKey' }}
+              href={`job-detail.html?username=${userName}&jobName=${jobName}`}
+            >
+              Go to Job Detail
+            </ActionButton>
+          </div>
+          <Stack horizontal gap='s1' padding='s2'>
+            <Text variant='large'>Job Name:</Text>
+            <Text variant='large'>{jobName}</Text>
+          </Stack>
+          <Stack horizontal gap='s1' padding='s2'>
+            <Text variant='large'>Job Attempt Index:</Text>
+            <Text variant='large'>{jobAttemptIndex}</Text>
+          </Stack>
+          <Stack horizontal gap='l1' padding='s2'>
+            <Stack gap='m'>
+              <Text>Task Role</Text>
+              <Text>{taskRoleName}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Index</Text>
+              <Text>{taskIndex}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Uid</Text>
+              <Text>{taskStatus.taskUid}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task State</Text>
+              <StatusBadge status={capitalize(taskStatus.taskState)} />
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Retries</Text>
+              <Text>{taskStatus.retries}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Start Time</Text>
+              <Text>
+                {DateTime.fromMillis(taskStatus.createdTime).toLocaleString(
+                  DateTime.DATETIME_MED,
```

Comment:
>DATETIME_MED [](start = 27, length = 12)

Align with job, use printDateTime

Reply:
Same reason, if we only want to print datetime format, then printDateTime is not needed.